docs/extras/modules/data_connection/vectorstores/integrations/meilisearch.ipynb

```diff
@@ -68,87 +91,60 @@
     "import os\n",
     "import getpass\n",
     "\n",
+    "# Prompt for Meilisearch credentials\n",
+    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")\n",
+    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")\n",
+    "\n",
+    "# Prompt for other third-party credentials\n",
```

Comment:
```suggestion
```
Same for this one

Reply:
Same answer, and I think having two examples makes it clearer that you can use the same technique for any environment variables.

Again, I'm not a Python developer so maybe this really is too basic to be explained. But langchain is so popular that there's likely a lot of non-Python devs using it.